.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: node_js
```

Comment:
maybe a different PR for this

Reply:
Could you revert the .travis.yml as it was before.

Reply:
that was provided as a solution, to the Travis problem